vips/vip-501/bsctestnet.ts

```diff
@@ -0,0 +1,329 @@
+        signature: "setTokenConfig((address,address[3],bool[3],bool))",
+        params: [
+          [
+            wstETH.address,
+            [WSTETH_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+            false,
+          ],
+        ],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+
+      // Market configurations
+      {
+        target: wstETHMarket.vToken.address,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["86400"],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+      {
+        target: wstETH.address,
+        signature: "faucet(uint256)",
+        params: [wstETHMarket.initialSupply.amount],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+      {
+        target: wstETH.address,
+        signature: "approve(address,uint256)",
+        params: [unichainsepolia.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+      {
+        target: wstETH.address,
+        signature: "approve(address,uint256)",
+        params: [unichainsepolia.POOL_REGISTRY, wstETHMarket.initialSupply.amount],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+
+      {
+        target: unichainsepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            wstETHMarket.vToken.address,
+            wstETHMarket.riskParameters.collateralFactor, // CF
+            wstETHMarket.riskParameters.liquidationThreshold, // LT
+            wstETHMarket.initialSupply.amount, // initial supply
+            unichainsepolia.NORMAL_TIMELOCK, // vToken receiver
+            wstETHMarket.riskParameters.supplyCap, // supply cap
+            wstETHMarket.riskParameters.borrowCap, // borrow cap
+          ],
+        ],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+      {
+        target: wstETHMarket.vToken.address,
+        signature: "transfer(address,uint256)",
+        params: [ethers.constants.AddressZero, wstETHMarket.initialSupply.vTokensToBurn],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+      (() => {
+        const vTokensMinted = convertAmountToVTokens(
+          wstETHMarket.initialSupply.amount,
+          wstETHMarket.vToken.exchangeRate,
+        );
+        const vTokensRemaining = vTokensMinted.sub(wstETHMarket.initialSupply.vTokensToBurn);
+        return {
+          target: wstETHMarket.vToken.address,
+          signature: "transfer(address,uint256)",
+          params: [wstETHMarket.initialSupply.vTokenReceiver, vTokensRemaining],
+          dstChainId: LzChainId.unichainsepolia,
+        };
+      })(),
+      {
+        target: wstETHMarket.vToken.address,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [wstETHMarket.riskParameters.protocolSeizeShare],
+        dstChainId: LzChainId.unichainsepolia,
+      },
```

Comment:
Unnecessary, because the current value is already correct

Reply:
https://github.com/VenusProtocol/vips/pull/561/commits/a307d272d9a8bb706bf64850057229e97c421d4b